.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Ramizjon  - I would suggest to exclude these eclipse-based files from the project, the same about ```.project```,```.settings/*```